homeassistant/components/yale_smart_alarm/strings.json

```
@@ -0,0 +1,21 @@
+{
+    "config": {
+      "abort": {
+        "already_configured": "[%key:common::config_flow::abort::already_configured_account%]"
+      },
+      "error": {
+        "auth_error": "[%key:common::config_flow::error::invalid_auth%]",
```

Please use our common string key names.
```suggestion
        "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
```

---

Thanks, I have changed